### DISCUSSION TITLE
EvergreenLibraryUpdate - Folder location

### DISCUSSION THREAD

**Discussion by cburbs** [EUCPilots/evergreen-module#618] [Q&A]
When using the Invoke-EvergreenLibraryUpdate is there any way to change Folder download? Example
In the JSON file I have
{
                             "Name":  "Zoom",
                             "EvergreenApp":  "Zoom",
                             "Filter":  "$_.Architecture -eq \"x64\" -and $_.Installer -eq \"Admin\" -and $_.Platform -eq \"Desktop\""
                         }

It saves to Evergreen\Zoom\5.17.5.31030\x64.

I would prefer it to save to Evergreen\Zoom or  Evergreen\Zoom\5.17.5.31030

I don't need the x64 folder.

**Comment by aaronparker** [comment]
Not in the current version - Evergreen does this so that multiple architectures can be supported.
